clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -64,16 +66,24 @@ public class ConsumerNetworkThreadTest {
     private final CompletableEventReaper applicationEventReaper;
 
     ConsumerNetworkThreadTest() {
+        LogContext logContext = new LogContext();
+        this.time = new MockTime();
+        this.applicationEventsQueue = new LinkedBlockingQueue<>();
         this.networkClientDelegate = mock(NetworkClientDelegate.class);
-        this.requestManagers = mock(RequestManagers.class);
         this.offsetsRequestManager = mock(OffsetsRequestManager.class);
         this.heartbeatRequestManager = mock(ConsumerHeartbeatRequestManager.class);
         this.coordinatorRequestManager = mock(CoordinatorRequestManager.class);
         this.applicationEventProcessor = mock(ApplicationEventProcessor.class);
         this.applicationEventReaper = mock(CompletableEventReaper.class);
-        this.time = new MockTime();
-        this.applicationEventsQueue = new LinkedBlockingQueue<>();
-        LogContext logContext = new LogContext();
+        this.requestManagers = spy(new RequestManagers(logContext,
```

Comment:
Since we directly call `requestManagers.consumerMembershipManager` in `ConsumerNetworkThread#closeInternal`, I replaced `mock` with `spy` to avoid an NPE.